cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java

```diff
@@ -413,6 +419,39 @@ private void diff(@CommandLine.Option(names = {"-s", "--source"}) String sourceN
         }
     }
 
+    @CommandLine.Command(name = "drivers", description = "Show available drivers for download", mixinStandardHelpOptions = true)
+    private void drivers(@CommandLine.Option(names = {"--list"}, description = "Used to list all available drivers") boolean isList,
+                         @CommandLine.Option(names = {"--download"}, description = "Used to download selected driver by index") boolean isDownload,
```

Comment:
Do you think we need a shortcut for `--download` ?

Reply:
Yes, I agree, should we add `-dl` ?